src/FunctionLoader.ts

```diff
@@ -15,13 +16,31 @@ export class FunctionLoader implements IFunctionLoader {
         info: FunctionInfo,
         func: Function
     }} = {};
+    private allowESModules = process.version.startsWith("v14");
 
-    load(functionId: string, metadata: rpc.IRpcFunctionMetadata): void {
+    async load(functionId: string, metadata: rpc.IRpcFunctionMetadata): Promise<void> {
       if (metadata.isProxy === true) {
           return;
       }
       let scriptFilePath = <string>(metadata && metadata.scriptFile);
-      let script = require(scriptFilePath);
+      let script: any;
+      if (scriptFilePath.endsWith(".mjs")) {
+        if (this.allowESModules) {
+          // IMPORTANT: pathToFileURL is only supported in Node.js version >= v10.12.0
+          // @ts-ignore
+          let scriptFileUrl = url.pathToFileURL(scriptFilePath);

```

Comment:
"import" takes a file URL and not a path... this behavior only shows up on windows though